Tests/SentryTests/Integrations/SentryCrash/SentryCrashIntegrationTests.swift

```diff
@@ -328,7 +328,7 @@ class SentryCrashIntegrationTests: NotificationCenterTestCase {
     }
     
     private func givenSutWithGlobalHubAndCrashWrapper() -> (SentryCrashIntegration, SentryHub) {
-        let sut = fixture.getSut(crashWrapper: SentryCrashWrapper.sharedInstance())
+        let sut = fixture.getSut(crashWrapper: SentryCrashWrapper()) // ???: should this be using TestSentryCrashWrapper?
```

Comment:
Question here. At some point we should also split out a separate test target for integration tests, which do not use mock subclasses of live SDK components.

Reply:
Yes, that could make sense. If the test succeeds with `TestSentryCrashWrapper`, please use it.